python/triton/runtime/jit.py

```diff
@@ -31,18 +31,27 @@ def get_cuda_stream(idx=None):
 def get_current_device():
     import torch
 
+    ignore_missing_cuda_device = os.environ.get("IGNORE_MISSING_CUDA_DEVICE", 'not found') in ["on", "true", "1"]
+    if ignore_missing_cuda_device and not torch.cuda.is_available():
+      return 0
     return torch.cuda.current_device()
 
 
 def set_current_device(idx):
     import torch
 
+    ignore_missing_cuda_device = os.environ.get("IGNORE_MISSING_CUDA_DEVICE", 'not found') in ["on", "true", "1"]
+    if ignore_missing_cuda_device and not torch.cuda.is_available():
+      return 0
```

Comment:
nitpick: this function does not return value.

P.s. ignore my review, I am not an maintainer, just passed by